actions/lib/actions.py

```diff
@@ -31,6 +32,6 @@ def _get_client(self):
         username = self.config['username']
         password = self.config['password']
 
-        client = auth.CFMClient(ipaddress, username, password)
+        client = CFMClient(ipaddress, username, password)
```

Comment:
cfm = client.CFMClient(...
then use cfm instead of client; otherwise the variable 'client' may conflict with the imported module 'client'